src/NexusMods.App.UI/Services.cs

```diff
@@ -182,9 +179,14 @@ public static IServiceCollection AddUI(this IServiceCollection c)
             .AddView<ApplyDiffView, IApplyDiffViewModel>()
             .AddView<FileTreeView, IFileTreeViewModel>()
             
-            .AddView<SettingsView, ISettingsViewModel>()
+            .AddView<SettingsView, ISettingsPageViewModel>()
+            .AddViewModel<SettingsPageViewModel, ISettingsPageViewModel>()
             .AddView<SettingEntryView, ISettingEntryViewModel>()
+            .AddViewModel<SettingEntryViewModel, ISettingEntryViewModel>()
             .AddView<SettingToggleControl, ISettingToggleViewModel>()
+            .AddViewModel<SettingToggleViewModel, ISettingToggleViewModel>()
+            .AddView<SettingComboBoxView, ISettingComboBoxViewModel>()
+            .AddViewModel<SettingComboBoxViewModel, ISettingComboBoxViewModel>()
```

Comment:
There is a View and VewModel separation in this file. Either stick to that or reorganize the file to be by semantic groups

Reply:
I hate the View/ViewModel groups in this file. It's much better to group by the actual type. I won't be changing the entire file.